Libs/MRML/Logic/vtkMRMLColorLogic.h

```diff
@@ -125,6 +176,118 @@ class VTK_MRML_LOGIC_EXPORT vtkMRMLColorLogic : public vtkMRMLAbstractLogic
   /// procedural node with a blank color transfer function.
   static vtkMRMLProceduralColorNode* CopyProceduralNode(vtkMRMLColorNode* colorNode, const char* copyName);
 
+  /// Return the label's terminology in this color table.
+  bool LookupCategorizationFromLabel(int label, ColorLabelCategorization&, const char *lutName = "GenericAnatomyColors");
+  /// Returns a string with the terminology term requested, or an empty string if not found.
```

Comment:
And to match what is done in MRMLScene: May be the name `GetCategorizationByLabel` ?